docs/source/quickstart.md

```diff
@@ -0,0 +1,242 @@
+```{currentmodule} batconf
+```
+
+```{toctree}
+:hidden: true
+:maxdepth: 2
+```
+
+# Quick Start
+
+## Install
+
+Install with `pip`:
+
+```bash
+pip install batconf
+```
+
+
+## Project Setup
+BatConf is designed to be minimally invasive.  Most projects only need a single
+file which imports from batconf.  Config classes are stdlib 
+[dataclass's](https://docs.python.org/3/library/dataclasses.html) 
+which can be used with or without BatConf.
+
+1. Add a {ref}`GlobalConfig` dataclass to your module's root `__init__.py`
+   1. The Config dataclasses provide basic config objects for your classes and 
+      modules.
+   2. Config dataclasses can be nested to create a configuration tree structure.
+2. Add the {ref}`get_config` function to `{yourmodule}/config.py`
```

Comment:
change to:
Add {ref}`get_config` function...

Reply:
fixed